package.json

```diff
@@ -1,21 +1,26 @@
 {
-  "name": "node-red-contrib-snmp-trap-listener",
-  "version": "1.3.0",
-  "description": "Node-RED node that receives snmp traps.",
-  "main": "index.js",
-  "repository": "https://github.com/ItsEcholot/node-red-contrib-snmp-trap-listener.git",
-  "author": "Marc Berchtold <marc.berchtold@hotmail.de>",
-  "license": "MIT",
-  "keywords": [
-    "node-red"
-  ],
-  "node-red": {
-    "nodes": {
-      "snmp-trap-listener": "snmp-trap-listener.js"
-    }
-  },
-  "dependencies": {
-    "moment": "^2.18.1",
-    "snmpjs": "git+https://github.com/joyent/node-snmpjs.git"
-  }
+	"name": "node-red-contrib-snmp-trap-listener",
+	"version": "2.0.0",
+	"description": "Node-RED node that receives snmp traps.",
+	"main": "index.js",
+	"repository": "https://github.com/ItsEcholot/node-red-contrib-snmp-trap-listener.git",
+	"dependencies": {
+		"moment": "^2.18.1",
+		"net-snmp": "^2.6.4"
+	},
+	"keywords": [
+		"node-red",
+		"snmp",
+		"trap"
+	],
+	"node-red": {
+		"nodes": {
+			"snmp-trap-listener": "snmp-trap-listener.js"
+		}
+	},
+	"author": [
+		"Marc Berchtold https://github.com/ItsEcholot/",
+		"Christophe Icard https://github.com/Ekristoffe/"
+	],
```

Comment:
Based on the package.json doc (https://docs.npmjs.com/files/package.json#people-fields-author-contributors) there can only be one author. Please add yourself to the contributors array.

Reply:
done